lib/timeline/component/ItemSet.js

```diff
@@ -1063,7 +1088,7 @@ ItemSet.prototype._onAddGroups = function(ids) {
   ids.forEach(function (id) {
     var groupData = me.groupsData.get(id);
     var group = me.groups[id];
-
+   
```

Comment:
Pedantic overdrive: spaces in an empty line.
*(I might be going over the top here, but I really do intend to review to the best of my capabilities)*